libr/util/w32-sys.c

```diff
@@ -27,6 +27,45 @@ static char *getexe(const char *str) {
 	return argv0;
 }
 
+R_API ut32 r_sys_get_winver() {
```

Comment:
what about making this funcction generic and make it work with uname on unix systems too instead of being windows-specific

Reply:
@radare I implemented r_sys_get_osinfo(). Can you check to see if thats what you wanted?